src/Apps/Authentication/Hooks/__tests__/useAuthDialogOptions.jest.tsx

```diff
@@ -0,0 +1,202 @@
+import { ContextModule, Intent } from "@artsy/cohesion"
+import { renderHook } from "@testing-library/react-hooks"
+import { useAuthDialogOptions } from "Apps/Authentication/Hooks/useAuthDialogOptions"
+import { useAuthDialogContext } from "Components/AuthDialog/AuthDialogContext"
+import { useRouter } from "System/Router/useRouter"
+
+jest.mock("System/Router/useRouter", () => ({
+  useRouter: jest.fn().mockImplementation(() => {
+    return { match: { location: { query: {} } } }
+  }),
+}))
+
+jest.mock("Components/AuthDialog/AuthDialogContext", () => ({
+  ...jest.requireActual("Components/AuthDialog/AuthDialogContext"),
+  useAuthDialogContext: jest.fn().mockImplementation(() => {
+    return { state: { options: {} } }
+  }),
+}))
+
+describe("useAuthDialogOptions", () => {
+  it("sets and returns defaults", () => {
+    const dispatch = jest.fn()
+
+    ;(useAuthDialogContext as jest.Mock).mockImplementation(() => ({
```

Comment:
can we push these into `mockUseAuthDialogContext` etc variables so they don't keep getting 🍝 'd?